lib/api/2.0/workflowGraphs.js

```diff
@@ -0,0 +1,67 @@
+// Copyright 2016, EMC Inc.
+
+'use strict';
+
+var injector = require('../../../index.js').injector;
+var controller = injector.get('Http.Services.Swagger').controller;
+var workflowApiService = injector.get('Http.Services.Api.Workflows');
+var _ = require('lodash');    // jshint ignore:line
```

Comment:
Since `lodash` has already in the injectable tables, so how about using the `injector.get` instead of require it directly?

I am thinking about a possible case to reduce the RackHD footprint in future. If the lodash have already in the on-core's injectable tables, so other modules (on-http, on-taskgraph, on-tasks, ...) can always obtain it via on-core, thus the `lodash` can be removed from on-http's dependencies, avoid to install a duplicated lodash in on-http. 

My point is that if the module has been included in on-core, then we should leverage the on-core to load it.